lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1267,7 +1276,17 @@ public FoundHits searchObjects(String objectType, MatchFilter matchFilter,
     private List<Map<String, Object>> prepareMatchFilters(MatchFilter matchFilter) {
         List<Map<String, Object>> ret = new ArrayList<>();
         if (matchFilter.fullTextInAll != null) {
-            ret.add(createFilter("match", "_all", matchFilter.fullTextInAll));
+            LinkedHashMap<String, Object> query = new LinkedHashMap<String, Object>() {{
```

Comment:
Is createFilter used anywhere else, and so should the changes go into createFilter rather than bypassing it? Is there anything in createFilter that the new code is not doing but should be?

Reply:
b/c createFilters expects everything to be singly nested, something like <a :{b:c}> and I want something like <a:{b:c, d}>

Reply:
The question is whether create filter should be changed to the new behavior rather than bypassing it.

Reply:
maybe. Its not being consistently used in the rest of the methods either.

Reply:
it may require a bunch of small changes everywhere though...